src/sql/workbench/parts/notebook/models/cell.ts

```diff
@@ -481,7 +481,10 @@ export class CellModel implements ICellModel {
 			if (serverInfo && serverInfo.options && serverInfo.options['clusterEndpoints']) {
 				let endpoints: notebookUtils.IEndpoint[] = serverInfo.options['clusterEndpoints'];
 				if (endpoints && endpoints.length > 0) {
-					endpoint = endpoints.find(ep => ep.serviceName.toLowerCase() === 'knox');
+					endpoint = endpoints.find(ep => {
```

Comment:
@chlafreniere can we track the need to move this code out? It is annoying that we have these special cases all over.
Gene - no change needed for you, just want it tracked for future Notebook work. Thanks!